.editorconfig

```diff
@@ -14,6 +14,87 @@ indent_size = 4
 [project.json]
 indent_size = 2
 
+# C# files
+[*.cs]
```

Comment:
@mellinoe, could you confirm that the convention is to break the defaults for `csharp_preserve_single_line_blocks` and `csharp_preserve_single_line_statements`, that is they should be false (single line statements and blocks should not be preserved)